developers/command-api.rst

```diff
@@ -0,0 +1,169 @@
+The Command API
+==========================
+
+What better way to expand your plugins using commands? This page will show how
+to create, register and unregister commmands.
+
+
+Create the command class
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Each command class implements the `Command interface <https://github.com/VelocityPowered/Velocity/blob/master/api/src/main/java/com/velocitypowered/api/command/Command.java>`_, which has two methods:
+one for when the command is executed and one to provide suggestions for tab completion.
+Let's see an example of a simple command that will tell whoever executes the command
+"Hello World" in light blue text.
+
+.. code-block:: java
+
+    package com.example.velocityplugin;
+
+    import com.velocitypowered.api.command.Command;
+    import com.velocitypowered.api.command.CommandSource;
+    import net.kyori.text.TextComponent;
+    import net.kyori.text.format.TextColor;
+    import org.checkerframework.checker.nullness.qual.NonNull;
+
+    public class CommandTest implements Command {
+
+        @Override
+        public void execute(@NonNull CommandSource source, String[] args) {
+            source.sendMessage(TextComponent.of("Hello World!").color(TextColor.AQUA));
+        }
+    }
+
+Now that we have created the command we need to register it in order for it to work.
```

Comment:
`Now that we have created the command, we need to register it in order for it to work.`